### PR TITLE
Problem: Typos in BEPs and project README

### PR DESCRIPTION
**Pull request by muawiakh**
:police_car: :police_car: Solution: Fix the typos :police_car: :police_car:

### CODE REVIEW

**Comment by vrde** [comment]
image

**Comment by ttmc** [comment]
It seems that BEP-4 is the one to standardize Docker Compose (DC), so I'll change the Illegal Data Response Plan to BEP-5. No problem. Too bad about the duel.

File: 3/README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Note: `"power": 0` implies that the validator should be removed from the validat
 
 
 ## Proposed Change
-- Add a new subcommand `bighciandb upsert-validator PUBLIC_KEY POWER` which will allow the node operator to add/update/delete a validator.
+- Add a new subcommand `bigchaindb upsert-validator PUBLIC_KEY POWER` which will allow the node operator to add/update/delete a validator.
```
Comment (ttmc) [inline 3/README.md]: Awesome.

**Comment by ttmc** [comment]
Something is _seriously up_ with GitHub's reviews. I tried to do one just now and there was no three-option bullet list (with Approve being one of them). Weird. Maybe just connectivity issues?

**Comment by ttmc** [comment]
I don't know how to change my review to "Approved." UI fail. Anyway, I approve this PR.

**Comment by shahbazn** [comment]
@ttmc i suppose thats because its merged already :)

**Pull request merged**